woocommerce-stats-by-currency.php

```diff
@@ -124,6 +124,70 @@ function wsbc_add_stats() {
 			<?php
 		}
 	}
-	
+
 }
 add_action( 'woocommerce_after_dashboard_status_widget', 'wsbc_add_stats' );
+
+/**
+ * Check required plugins
+ *
+ * @return void
+ */
+
+function wsbc_required_plugins() {
+	$activated = (array) get_option( 'active_plugins', array() );
+	$required  = array(
+		'woocommerce/woocommerce.php',
+		'woocommerce-product-price-based-on-countries/woocommerce-product-price-based-on-countries.php'
+	);
+
+	$required_plugins_activated = count( array_intersect($activated, $required) ) == count( $required );
+
+    if ( !$required_plugins_activated  ) { ?>
+   
+        <?php
+        $style          = '<style type="text/css">
+                                .wp-die-message {
+                                    border: 1px solid #ccd0d4;
+                                    border-left: solid 5px #dc3232;
+                                    padding: 12px 0 12px 25px;
+                                }
```

Comment:
let's simplify this for better design & style. moving border to body/ error wrapper

```suggestion
                                #error-page {
                                    border-left: solid 5px #dc3232;
                                }
```

<img width="827" alt="die_msg" src="https://user-images.githubusercontent.com/1039236/80305729-3af65b00-87dc-11ea-962a-d0402b6b2579.png">